src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -277,6 +276,41 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 			DOM.getContentWidth(this.codeElement.nativeElement),
 			DOM.getContentHeight(this.codeElement.nativeElement)));
 		this._editor.setHeightToScrollHeight(false, this._cellModel.isCollapsed);
+		this.horizontalScrollbar();
+	}
+
+	public horizontalScrollbar(): void {
+		if (this._configurationService.getValue('editor.wordWrap') === 'off' && this.cellModel.cellType !== CellTypes.Code) {
+			// Get markdown split view horizontal scrollbar
+			let horizontalScrollbar = this.codeElement.nativeElement.querySelector('div.invisible.scrollbar.horizontal');
+
+			// Compare the editor bottom position to the scrollable area in order to decide whether to make the horizontal scrollbar fixed or absolute
+			let editorBottomPos = this.codeElement.nativeElement.closest('.show-markdown .editor').getBoundingClientRect().bottom - this.codeElement.nativeElement.closest('.show-markdown .editor').getBoundingClientRect().top;
+			let viewportHeight = DOM.getTotalHeight(document.querySelector('.scrollable'));
+
+			// Horizontal scrollbar values used when fixed
+			let viewportTop = document.querySelector('.scrollable').getBoundingClientRect().top;
+			let horizontalTop = viewportTop + viewportHeight - horizontalScrollbar.scrollHeight - 10;
+
+			// Set the bottom position of the markdown editor area of the active cell
+			let toolbarOffsetHeight = DOM.getContentHeight(document.querySelector('markdown-toolbar-component')) + DOM.getContentHeight(document.querySelector('cell-toolbar-component'));
+			editorBottomPos = editorBottomPos + toolbarOffsetHeight + horizontalScrollbar.scrollHeight;
+
+			// When the editor of the cell is past the scrollable area we will change the horizontal scrollbar to be fixed to the bottom of the visible area (viewport)
+			if (editorBottomPos >= viewportHeight) {
+				horizontalScrollbar.style.opacity = 1;
+				horizontalScrollbar.style.position = 'fixed';
+				horizontalScrollbar.style.left = '';
+				horizontalScrollbar.style.top = horizontalTop + 'px';
+			} else {
+				// When scrollbar is not fixed and cell is still in visible area
+				horizontalScrollbar.style.opacity = 1;
```

Comment:
Initial scrollbar styles should keep it hidden until called for.

Reply:
Agreed this would fix when there is no text in the cell.